chimera_ctioenviroment/instruments/lcogtweather.py

```diff
@@ -34,38 +34,35 @@ class LCOGTScrapper(object):
     Web scrapper for the LCOGT telops page
     """
 
-    def scrape(self):
-        client = requests.session()
+    def __init__(self):
+        self.results = dict()
 
-        data = client.get('https://telops.lco.global/#', timeout=60)
+    def scrape(self):
 
-        try:
-            latest_comet_queue_id = int(re.findall('Telops.latest_comet_queue_id = (.+);', data.text)[0])
-        except IndexError:
-            return None
-
-        r = client.post(
-            url='https://telops.lco.global/dajaxice/netnode.refresh/',
-            data={'argv': json.dumps({"latest": latest_comet_queue_id})},
-            headers={
-                'Accept': '*/*',
-                'Accept-Encoding': 'gzip, deflate',
-                "Content-Type": "application/x-www-form-urlencoded",
-                'Host': 'telops.lco.global',
-                "Origin": "https://telops.lco.global/",
-                "Referer": "https://telops.lco.global/",
-                'X-CSRFToken': None,
-                'X-Requested-With': 'XMLHttpRequest',
-
-            },
-            cookies={'pushstate': 'pushed'},
-            timeout=30
-        )
-        return json.loads(r.text)
-
-
-class LCOGTWeather(WeatherBase, WeatherTemperature, WeatherHumidity, WeatherPressure, WeatherWind, WeatherTransparency,
-                   WeatherSafety):
+        _base_url = "https://weather-api.lco.global/query?site=lsc&datumname="
+        urls = {'humidity': _base_url + "Weather%20Humidity%20Value",
+                'temperature': _base_url + "Weather%20Air%20Temperature%20Value",
+                'wind_speed': _base_url + "Weather%20Wind%20Speed%20Value",
+                'wind_direction': _base_url + "Weather%20Wind%20Direction%20Value",
+                'dew_point': _base_url + "Weather%20Dew%20Point%20Value",
+                'pressure': _base_url + "Weather%20Barometric%20Pressure%20Value",
+                'sky_transparency': _base_url + "Boltwood%20Transparency%20Measure"
+                }
+
+        for key in urls.keys():
+            try:
+                self.results[key] = requests.get(urls[key]).json()[-1]
+            except:
+                self.results[key] = {u'TimeStamp': u'',
+                                     u'TimeStampMeasured': u'',
+                                     u'Value': None,
+                                     u'ValueString': u''}
+
+        return self.results
```

Comment:
why do you need result as `self.result`?

Reply:
I don't. Just a donkey programmer thing.